web/source/kmwapi.ts

```diff
@@ -0,0 +1,44 @@
+/// <reference path="closure.ts" />
+/// <reference path="kmwbase.ts" />
+/// <reference path="kmwutils.ts" />
+
+// Allows proper minification handling.
+/// <reference path="../node_modules/google-closure-library/closure/goog/base.js" />
+
+/**
+ * This file provides calls needed to export our established API elements when performing minification.
+ * Closure can work with these instructions to prevent it from eliminating these names.
```

Comment:
Noting the new issue #518, this file exists to provide those 'extern' declarations and ensure they're preserved during minification.